new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/TestUtils.java

```diff
@@ -58,6 +59,10 @@ public static boolean callWebAppAndCheckForServerNameInResponse(
           });
         } catch (Exception e) {
           logger.info("Got exception while running command: " + curlCmd);
+          if (result != null) {
+            logger.info("result.stdout: \n {0}", result.stdout());
```

Comment:
get rid of the extra space after `\n` to avoid confusion

Reply:
fixed